scipy/signal/tests/test_signaltools.py

```diff
@@ -534,6 +534,11 @@ def test_basic(self):
         assert_raises(ValueError, signal.resample_poly, sig, 'yo', 1)
         assert_raises(ValueError, signal.resample_poly, sig, 1, 0)
 
+        # test for issue #6505 - should not modify window.shape when axis ≠ 0
```

Comment:
Travis error:

https://travis-ci.org/scipy/scipy/jobs/154339732#L3379

You either need to get rid of the non-ASCII character, or add the utf-8 header `# -*- coding: utf-8 -*-` to the top of the file